src/main/java/net/querz/mcaselector/io/mca/MCAFile.java

```diff
@@ -450,16 +450,13 @@ protected void mergeChunksInto(MCAFile<T> destination, Point3i offset, boolean o
 					}
 
 					if (ranges != null) {
-						int sourceVersion = sourceChunk.getData().getInt("DataVersion");
-						if (sourceVersion == 0) {
-							continue;
-						}
+						int sourceVersion = sourceChunk.getDataVersion();
```

Comment:
This now will cancel the merging process of the entire file / all files if it throws the `NoSuchElementException`, as opposed to before where it simply ignored it.

Reply:
Thinking about this, we'll need to look into all other places where this might be an issue